helm-charts/0.25.4/charts/keystone/templates/Deployment.yaml

```diff
@@ -62,6 +62,7 @@ spec:
             #
           {{- $safeInitEndpointUrlSet := false }}
           {{- $safeInitSpiffeIdPrefixSet := false }}
+          {{- $workloadInitSpiffeIdPrefixSet := false }}
```

Comment:
workload spiffeid prefixes need to be configurable too.

Long story, but mostly because OpenShift, or more properly because certain OpenShift configurations.